### PR TITLE
fix(logs): align pod name truncation

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-1168

2024-07-01-104521_908x506_scrot

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> You are going to thank you later if it contains tests

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [\[PR\] test-remi2 - fix(logs): align pod name truncation - 2024-07-01T08:47:54Z](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/e773b890-e6b2-4406-93a9-71f9a7ab6d73/environment/ae37107f-e901-458a-82ba-dc1addcd3c57/services/general)

 Another comment will be posted when deployments are finished

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Your preview environment failed to be deployed !
Click on the link below to check the deployment logs.
:point_right: [\[PR\] test-remi2 - fix(logs): align pod name truncation - 2024-07-01T08:47:54Z](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/e773b890-e6b2-4406-93a9-71f9a7ab6d73/environment/ae37107f-e901-458a-82ba-dc1addcd3c57/logs)

File: libs/pages/logs/environment/src/lib/ui/pod-logs/pod-logs.tsx [hunk 1]
```diff
@@ -103,7 +103,11 @@ export function PodLogs({
                 <div className="mr-2.5 w-4">
                   <StatusChip status={currentPod?.state} />
                 </div>
-                <p className="mr-5 truncate text-xs font-medium text-neutral-100">{data.pod_name}</p>
+                <p className="mr-5 truncate text-xs font-medium text-neutral-100" title={data.pod_name}>
+                  {data.pod_name && data.pod_name.length > 23
+                    ? `${data.pod_name?.substring(0, 10)}...${data.pod_name?.slice(-10)}`
```
Comment (RemiBonnet) [inline libs/pages/logs/environment/src/lib/ui/pod-logs/pod-logs.tsx:108]: We have an old helper to apply it `trimId` maybe can we custom it ?

Comment (ctjhoa) [reply libs/pages/logs/environment/src/lib/ui/pod-logs/pod-logs.tsx]: Here we go :
https://github.com/Qovery/console/pull/1460/commits/b9f04552d041d10401685d7717bb210b987fbbac

**Review by RemiBonnet — APPROVED** [review]

**Pull request merged**